src/libsyncengine/syncpal/operationprocessor.h

```diff
@@ -44,7 +44,11 @@ class OperationProcessor : public ISyncWorker {
          * @return a shared pointer to the node in other tree. nullptr il not found.
          */
         std::shared_ptr<Node> correspondingNodeDirect(std::shared_ptr<Node> node);
+
         bool isABelowB(std::shared_ptr<Node> a, std::shared_ptr<Node> b);
+
+    private:
+        std::shared_ptr<Node> findFromCommonAncestor(std::shared_ptr<Node> node);
```

Comment:
```suggestion
        std::shared_ptr<Node> findClosestExistingDbAncestor(std::shared_ptr<Node> node);
```
Does this name reflects the intention? (I don't recommend the name I made up, but I am looking for a more satisfactory function name).

It would be nice to have a short text description about what this function does.

Reply:
Ok. Your proposition did not reflect the goal of the method accuratly (we are trying to find the corresponding node by looking for a common ancestor, we do not want to return the ancestor but the corresponding node).
I tried a more explicit name and added a short description. Let me know if it is clear enough now